src/java/org/apache/cassandra/batchlog/Batch.java

```diff
@@ -77,12 +79,17 @@ public int size()
     {
         return decodedMutations.size() + encodedMutations.size();
     }
-
-    static final class Serializer implements IVersionedSerializer<Batch>
+    
+    public boolean isLocal()
```

Comment:
We could add some brief JavaDoc telling what local means, similar to the one for the both `createLocal` and `createRemote` methods right above. By the way, `createRemote` has an unneeded `<Mutation>` that we could remove here.